.github/workflows/aws-eks-test.yml

```
@@ -80,7 +80,7 @@ jobs:
         if: always()
         # Add any artifacts that should be associated with this run to /tmp/artifacts/${{ github.run_id }}
         run: |
-          mkdir -p /tmp/artifacts/${{ github.run_id }}
+          mkdir -p /tmp/artifacts/${{ github.run_id }}/logs /tmp/artifacts/${{ github.run_id }}/resources
```

Nit: just to be fancy

```suggestion
          mkdir -p /tmp/artifacts/${{ github.run_id }}/{logs,resources}
```

---

🪄 Nice.